homeassistant/components/roborock/coordinator.py

```diff
@@ -168,6 +181,26 @@ async def set_current_map_rooms(self) -> None:
             for room in room_mapping or ()
         }
 
+    async def get_scenes(self) -> list[HomeDataScene]:
+        """Get scenes."""
+        return await self._api_client.get_scenes(
```

Comment:
`execute_scene` catches exceptions and raises home assistant error but this does not. Can these be consistent?

Reply:
sure